applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -842,6 +840,72 @@ static struct bt_conn_cb conn_callbacks = {
 
 };
 
+/**
+ * @brief	Callback handler for play/pause.
+ *
+ * @note	This callback is called from MCS after receiving a
+ *		command from the client or the local media player.
+ *
+ * @param[in]	play  Boolean to indicate if the stream should be resumed or paused.
+ */
+static void le_audio_play_pause_cb(bool play)
+{
+	int ret;
+
+	LOG_DBG("Play/pause cb, state: %d", play);
+
+	if (play) {
+		playing_state = true;
+
+		if (audio_streams[AUDIO_CH_L].ep->status.state ==
+		    BT_AUDIO_EP_STATE_QOS_CONFIGURED) {
+			ret = bt_audio_stream_enable(&audio_streams[AUDIO_CH_L],
+						     lc3_preset_nrf5340.codec.meta,
+						     lc3_preset_nrf5340.codec.meta_count);
+
+			if (ret) {
+				LOG_WRN("Failed to enable left stream");
+			}
+		}
```

Comment:
Should add warning (?) prints that streams is not in BT_AUDIO_EP_STATE_QOS_CONFIGURED state

Reply:
I did, but as we discussed we have to send a play command on the gateway to set the media player to playing state, that will cause warning prints here since the streams might not be enabled yet.